src/test/navbar.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom';
+import Navbar from '../components/navbar';
+
+test('Navbar matches snapshot', () => {
+  const component = renderer.create(
+    <MemoryRouter>
+      <Navbar />
+    </MemoryRouter>,
+  );
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] as part of the requirements : 

- The App should have unit tests for several components of the app and some pure functions in Redux code (currently you only tested the navbar component) 
- The App should have integration tests using React Testing Library

kindly add that 🙏